cli/Cargo.toml

```diff
@@ -91,10 +76,10 @@ vergen = { workspace = true, features = ["cargo"] }
 
 [package.metadata.cargo-all-features]
 denylist = [
-"bridge",
-"dex",
-"schema-endpoint",
-"telemetry",
-"test-network"
+  "bridge",
+  "dex",
```

Comment:
Oh, we should remove that confusing stuff someday

Reply:
Issued https://github.com/hyperledger/iroha/issues/4152